Yosemite/Yosemite/Actions/TaxClassesAction.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import Networking
+
+
+/// TaxClassesAction: Defines all of the Actions supported by the TaxClassesStore.
+///
+public enum TaxClassesAction: Action {
```

Comment:
nit: similar to the comment for the Yosemite store, I'd suggest the name `TaxClassAction`